test/bundle.c

```diff
@@ -157,6 +157,9 @@ static void bundle_test_extract_manifest(BundleFixture *fixture,
 	free_bundle(bundle);
 }
 
+// Hack to pull-in context for testing modification
+extern RaucContext *context;
```

Comment:
I'm not happy about this hack, but it's probably the easiest solution for the tests.

Reply:
Ok, then I would leave it as it is ;)